module1/main.c

```diff
@@ -0,0 +1,42 @@
+/* Module 1: ArrayList */
+#include <stdio.h>
+#include <stdlib.h>
+#include "mod1.h"
+
+char ACTIONS[] = "0\tQuit\n1\tPrint array\n2\tAdd to front of array\n";
+
+int main() {
+    printf("*** Module 1: ArrayList (integers only) ***\n");
+
+    // Initialize ArrayList
+    struct array_list* arrlst = malloc(sizeof(struct array_list));
```

Comment:
Does this mem need to be freed before end of program? Ditto for allocations in `mod1.c`